client/jetpack-connect/happychat-button.jsx

```diff
@@ -13,20 +13,30 @@ import { localize } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
+
+import analytics from 'lib/analytics';
 import HappychatButton from 'components/happychat/button';
 import HappychatConnection from 'components/happychat/connection';
 import { isEnabled } from 'config';
 import { getCurrentUserId } from 'state/current-user/selectors';
 import hasActiveHappychatSession from 'state/happychat/selectors/has-active-happychat-session';
 import isHappychatAvailable from 'state/happychat/selectors/is-happychat-available';
 
+const getHappyChatButtonClickHandler = eventName => {
+	eventName = eventName || 'calypso_jpc_chat_initiated';
```

Comment:
We could pass the default event name in the definition for brevity:

```
const getHappyChatButtonClickHandler = ( eventName = 'calypso_jpc_chat_initiated' ) => {
```

I'm fine with what you've suggested too, if that's what you prefer.